package.json

```diff
@@ -19,8 +19,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "alexa-app": "^2.4.0",
-    "alexa-verifier-middleware": "^0.1.8",
+    "alexa-app": "git+https://github.com/mreinstein/alexa-app.git#352f17cf9b8f541d8f32a385bfff61877b60134e",
```

Comment:
it was a necesary evil while developing/testing. My deploy environment caches deps aggressively. This absolutely should point at canonical npm version as soon as possible. same goes for entries in `examples/apps/*package.json`